compass/ocean/tests/global_ocean/files_for_e3sm/graph_partition.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 
 
-def get_core_list(ncells, max_cells_per_core=6000, min_cells_per_core=100):
+def get_core_list(ncells, max_cells_per_core=30000, min_cells_per_core=2):
```

Comment:
It seems like there are situations where folks really want to push the limits of how many or how few cells you can get away with.  I'm including some partitions that accommodate these needs.  We can back off, but it's at the risk that E3SM developers generate their own partitions (which is a little riskier with the new sea ice approach).